packages/wrangler/src/__tests__/helpers/msw/handlers/script.ts

```diff
@@ -29,46 +30,28 @@ export default [
 	rest.get(
 		"*/accounts/:accountId/workers/services/:scriptName/environments/:env/content",
 		({ params: { scriptName } }, res, context) => {
-			return res(
-				context.status(200),
-				context.text(getScript(scriptName as string))
-			);
+			return res(context.text(getScript(scriptName as string)));
```

Comment:
NIT: you could change `getScript()` to accept a `PathParams<string>` and do the cast to string in there, avoiding having to do it in every test.

Reply:
Done.